src/errors.js

```diff
@@ -0,0 +1,34 @@
+function checkError(originalText, targetCase) {
+  const validCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const errors = [];
+
+  if (!originalText) {
+    errors.push({
+      message:
+        'Text to convert is required. ' +
+        'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (!targetCase) {
+    errors.push({
+      message:
+        '"toCase" query param is required.' +
+        ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (targetCase && !validCases.some((cas) => cas === targetCase)) {
```

Comment:
This _if_ could be an _else_ of the previous _if_
```
if (!targetCase) {
    ...
} else if (!validCases ...) {
    ...
}
```

And another possible approach would be to change _some_ for _includes_, having the same result